test/parallel/test-require-symlink.js

```diff
@@ -1,53 +1,80 @@
 // Flags: --preserve-symlinks
 'use strict';
 const common = require('../common');
+
+const { execSync, spawn } = require('child_process');
+
+if (common.isWindows) {
+  // On Windows, creating symlinks requires admin privileges.
+  // We'll only try to run symlink test if we have enough privileges.
+  try {
+    const o = execSync('whoami /priv');
+    if (!o.includes('SeCreateSymbolicLinkPrivilege'))
+      common.skip('insufficient privileges');
+  } catch (e) {
+    common.skip('insufficient privileges');
+  }
+}
+
 const assert = require('assert');
-const path = require('path');
 const fs = require('fs');
-const { exec, spawn } = require('child_process');
+const path = require('path');
 const util = require('util');
+
+// Setup: Copy fixtures to tmp directory.
+
 const fixtures = require('../common/fixtures');
+const dirName = 'module-require-symlink';
+const fixtureSource = fixtures.path(dirName);
+const tmpDirTarget = path.join(common.tmpDir, dirName);
 
+// Copy fixturesSrouce to linkTarget recursively.
 common.refreshTmpDir();
 
-const linkTarget = fixtures.path('module-require-symlink',
-                                 'node_modules',
-                                 'dep2');
+function copyDir(source, target) {
```

Comment:
Wouldn't you rather `execSync('cp -Rf Src Trg')` / for windows `execSync('xcopy /E /Y Src Trg')`?

Reply:
I prefer the more verbose way here because:

* there's no difference between Windows and POSIX test code that way
* it does not depend on even the most mundane external dependency. 

`execSync` always feels like a code smell to me, although it's almost certainly not going to cause problems here.

I admit my reasons are not super-compelling but I'm also not sure the counter-arguments are super-compelling either. I'll certainly change it if this is a blocking objection. :-)